### DISCUSSION TITLE
Nested <svelvet> components support

### DISCUSSION THREAD

**Discussion by heaven-born** [open-source-labs/Svelvet#458] [General]
Hello. I'm considering moving my vuejs project to svelt because it has this great svelvet library that also seems to support nested canvas components. At least my initial experiments with the library look promising regarding this feature because it seems to be working at least in general. The only issue that I noticed was that the background grids of the parent component and nested component are somehow connected (if I zoom the nested component it also changes the size of the parent component grid). 

Can someone confirm that the nested components are supported officially or it just happened to be working accidentally and there is no official support for this feature? 

My application requires at least 3 levels of the nested canvas components.

**Comment by BOJIT** [comment]
I suspect the [scroll handle function](https://github.com/open-source-labs/Svelvet/blob/main/src/lib/containers/Graph/Graph.svelte#L386) is firing the parent when scrolling the nested grid.

I don't know if nested grids is an officially supported feature: (@briangregoryholmes ?), however I reckon that you could possibly fix with an `e.stopPropogation()` inside the `handleScroll` function. Perhaps worth a PR if it doesn't break any existing behaviour.
